### ISSUE TITLE
ckfinder in iframe - attach token

### ISSUE THREAD

**Issue by svkmedia** [ckfinder/ckfinder-laravel-package#25]
I'm trying to open CKFinder in iframe. Problem is that I have middleware checking routes for browser and connector. 
When I hit example.com/ckfinder/browser?token=123 everything works fine.
But when I pass exactly the same URL into iframe I get response: Please attach your Bearer token. I updated setup.php in vendor package like this:

`CKFinder.config( {
   connectorPath: <?php echo json_encode(route('ckfinder_connector')) ?>,
        pass: 'token',
        token: '<?php echo (isset($_GET['token'])) ? $_GET['token'] : "" ?>',
    } );`

but still not working...
Any advice how to make it work in iframe?

**Comment by zaak** [comment]
Hi @svkmedia! Could you please clarify what do you mean by opening in an `iframe`? What error exactly do you use? Do you create the `iframe` yourself, or you use the [widget mode](https://ckeditor.com/docs/ckfinder/demo/ckfinder3/samples/widget.html)?

If you use CKFinder inside a custom `iframe` element it may be a bit tricky to make CKFinder work properly because of cross-window communication restrictions. I suggest trying the widget mode instead - in this mode, CKFinder creates the iframe itself, so everything should work properly.

**Comment by svkmedia** [comment]
@zaak yes I create iframe manually. I was thinking about using widget but I have no idea how to use it because I'm trying to implement it into Vue/Laravel app.
I succesfully integrated ekeditor with ckfinder. Now I need button in app navigation to open iframe/widget of ckfinder.
Can I change browser.php and setup.php in vendor package? I'm afraid it will affect ckeditor finder...

**Comment by d4rkr3pt0r** [comment]
@svkmedia 
Here you can find all you need : 
https://github.com/ckfinder/ckfinder-laravel-package/issues/37#issuecomment-735046945
